### ISSUE TITLE
Users can't see "To Validate Requests"

### ISSUE THREAD

**Issue by carlosbmc** [pluginsGLPI/formcreator#2216]
The users creates a forms witch needs to be validated,  but can not see it.   
The count off "to validate" Forms is "0"

![image](https://user-images.githubusercontent.com/31730629/115232084-a2589980-a116-11eb-826f-791ade69570e.png)

**Comment by carlosbmc** [comment]
![image](https://user-images.githubusercontent.com/31730629/115232233-c3b98580-a116-11eb-8d54-23cf4cbfb332.png)

**Comment by btry** [comment]
Hi

The "to validate" counter includes form answers to be validated by the current user. Is the current user in charge of a validation step for those form answers ?

**Comment by carlosbmc** [comment]
No,  is the user who have created the request.   Can the user who have created the requests see it's own requests?

**Comment by btry** [comment]
Hi,

Yes, the requester shall see his own requests when he clicks on "my assistance requests" on the left panel.

**Comment by btry** [comment]
Hi

This PR may solve this issue. It requires to run an upgrade as the DB schema is changed.

https://github.com/pluginsGLPI/formcreator/pull/2536
